config/application.yml.example

```diff
@@ -19,7 +19,7 @@ EVSS_SAMPLE_CLAIMANT_USER: '{"uuid": "1234", "first_name": "Jane","last_name": "
 MVI_URL: 'http://ps-dev.commserv.healthevet.va.gov:8110/psim_webservice/IdMWebService'
 VHA_MAPSERVER_URL: 'https://maps.va.gov/server/rest/services/PROJECTS/Facility_Locator/MapServer'
 VHA_MAPSERVER_LAYER: '0'
-MOCK_MVI_SERVICE: true
+MOCK_MVI_SERVICE: 'true'
```

Comment:
env vars come through as strings and figaro gives a warning if you don't wrap in quotes